inmem.go

```diff
@@ -264,3 +284,14 @@ func (i *InmemSink) flattenKey(parts []string) string {
 	joined := strings.Join(parts, ".")
 	return strings.Replace(joined, " ", "_", -1)
 }
+
+// Flattens the key for formatting along with its labels, removes spaces
+func (i *InmemSink) flattenKeyLabels(parts []string, labels []Label) (string, string) {
+	key := strings.Replace(strings.Join(parts, "."), " ", "_", -1)
```

Comment:
If performance is a concern these things could be done more efficiently with a bytes buffer and https://golang.org/pkg/strings/#Replacer